src/app/pages/resources/overview/overview.jsx

```diff
@@ -411,36 +400,41 @@ function ChplResourcesOverview() {
               </TableRow>
             </TableHead>
             <TableBody>
-              { getOrgs(acbQuery, 'acbs').map((acb) => (
+              {getOrgs(acbQuery, 'acbs').map((acb) => (
                 <TableRow key={acb.id}>
                   <TableCell>ONC-ACB</TableCell>
                   <TableCell>{acb.name}</TableCell>
                   <TableCell>{acb.acbCode}</TableCell>
                   <TableCell>
                     {acb.website
-                     && <ChplLink href={acb.website} />}
+                      && <ChplLink href={acb.website} />}
                   </TableCell>
                 </TableRow>
               ))}
-              { getOrgs(atlQuery, 'atls').map((atl) => (
+              {getOrgs(atlQuery, 'atls').map((atl) => (
                 <TableRow key={atl.id}>
                   <TableCell>ONC-ATL</TableCell>
                   <TableCell>{atl.name}</TableCell>
                   <TableCell>{atl.atlCode}</TableCell>
                   <TableCell>
                     {atl.website
-                     && <ChplLink href={atl.website} />}
+                      && <ChplLink href={atl.website} />}
                   </TableCell>
                 </TableRow>
               ))}
             </TableBody>
           </Table>
-          <InternalScrollButton
-            name="Back to the top"
-            id="main-content"
-            analytics={{ event: 'Jump to top of Overview', category: 'Navigation' }}
-          />
-          <Divider />
+          <div >
+            <Fab
+              name="Back to the top"
+              className={classes.FabContainer}
+              color="primary"
+              id="main-content"
+              analytics={{ event: 'Jump to top of Overview', category: 'Navigation' }}
```

Comment:
`Fab` is a material-ui thing, right? `analytics` is an attribute I added and used specifically to send analytics when needed. I can't see how this could work

Reply:
Yes Fab is a material ui component. Am i not that well verse on using analytics I copied what you did on internal scroll button for analytics. I thought the action we are tracking is the same, maybe we add a label for "Back to the top"?